applications/ipc_radio/README.rst

```diff
@@ -76,12 +73,12 @@ The Bluetooth Low Energy and IEEE 802.15.4 functionalities can operate simultane
 .. note::
    The IEEE 802.15.4 is currently not supported on the :ref:`zephyr:nrf54h20dk_nrf54h20` board.
 
-Sysbuild
-========
+Sysbuild Kconfig options
+========================
 
-To enable the firmware, use the Sysbuild configuration of :kconfig:option:`SB_CONFIG_NRF_DEFAULT_IPC_RADIO`.
+To enable the firmware, use the sysbuild configuration of :kconfig:option:`SB_CONFIG_NRF_DEFAULT_IPC_RADIO`.
```

Comment:
this is for nordic sample applications only, customer applications should use `SB_CONFIG_NETCORE_IPC_RADIO`

Reply:
Created a task for @dchat-nordic : https://nordicsemi.atlassian.net/browse/NCSDK-27939